app/ai/text/api.py

```
@@ -0,0 +1,157 @@
+    try:
+        result = await agent_info["invoke_fn"](description=request.input)
+        elapsed = round(time.time() - start_time, 1)
+
+        return PlaygroundResponse(
+            agent=request.agent,
+            status=result.get("status", "unknown"),
+            message=result.get("message", ""),
+            result=result.get("result"),
+            elapsed_secs=elapsed,
+        )
+    except Exception as e:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)